test/support/utils.rb

```diff
@@ -42,12 +42,14 @@ def debug_print msg
 
     RUBY = RbConfig.ruby
 
-    def create_pseudo_terminal
+    def create_pseudo_terminal(start_with_session: true)
```

Comment:
or `boot_option: '-r require debug/run'`

Reply:
sure, I guess this will help testing other modes as well 👍